definitions/tools/cellmatch_lineage.cwl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Running a script to identify lineage of cells"
+
+baseCommand: ["/usr/local/bin/Rscript", "/opt/CellMatch_Haemopedia.r"]
+
+requirements:
+    - class: DockerRequirement
+      dockerPull: "saimukund/seurat_docker"
```

Comment:
Adding a version tag to this would be good

Reply:
Changes made!